spec/spec_helper_acceptance.rb

```diff
@@ -440,10 +470,19 @@ class BeakerRunnerBase
   include Beaker::DSL
   include MasterManipulator::Site
 
-  def remote_environment
+  def remote_environment(host)
+    if host['platform'] =~ /^windows/
```

Comment:
This could be written as

```ruby
azure_cert = host['platform'] =~ /^windows/ ? 'c:\azure_cert.pem' : LINUX_AZURE_CERT
```

Same comment as above about the missing else if you want to constrain this to only known platforms. Personally I'd use this formulation and leave the assumption.

Alsom it's odd that the linux variant is a CONSTANT and the windows var not. I'd choose one or the other so the information is in one place.